<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFTPlot</class>
 <widget class="QDialog" name="FFTPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FFTW result</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="3" colspan="3">
    <widget class="QCustomPlot" name="widget_A" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Reset PlotRange</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Fourier Filter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="9" column="0">
       <widget class="QPushButton" name="pushButton_ApplyFilter">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Apply  Filter</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_MovingAvg">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Moving Avg.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_CalFFT">
        <property name="text">
         <string>Fourier Forward</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButton_LowPassSharp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Low-Pass Sharp Filter</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Rectangle Filter</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_MovingAvg">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="radioButton_Guassian">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Guassian Filter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_RemoveConstant">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove fy= 0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButton_LowPass">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Low-Pass Filter</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButton_BandPassSharp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Band-Pass Sharp Filter</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>pushButton_ApplyFilter</zorder>
     <zorder>checkBox_MovingAvg</zorder>
     <zorder>pushButton_CalFFT</zorder>
     <zorder>radioButton_LowPassSharp</zorder>
     <zorder>radioButton</zorder>
     <zorder>spinBox_MovingAvg</zorder>
     <zorder>checkBox_RemoveConstant</zorder>
     <zorder>radioButton_LowPass</zorder>
     <zorder>radioButton_BandPassSharp</zorder>
     <zorder>radioButton_Guassian</zorder>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_freqL">
     <property name="text">
      <string>freqeuncy low</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lineEdit_freqL">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="7">
    <widget class="QLineEdit" name="lineEdit_Msg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSlider" name="horizontalSlider_freqL">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="3" colspan="3">
    <widget class="QCustomPlot" name="widget_P" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_freqH">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>freqeuncy high</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="horizontalSlider_freqH">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="lineEdit_freqH">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QPushButton" name="pushButton_FFTWBackward">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Fouier Backward</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
